mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
+CC_x86_64-unknown-linux-uclibc=$(CFG_UCLIBC_ROOT)/bin/uclibc-gcc
```

Comment:
Is there a way that I can error the build out if the variable `CC_x86_64-unknown-linux-uclibc` is not set in this file?